src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,19 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    public static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error("AuthenticationException " + e);
```

Comment:
```suggestion
            logger.error("Can't log in user: " + user + e);
```

Reply:
We can't add user because it is not initialised 
![image](https://user-images.githubusercontent.com/118391929/210872955-ba2138bc-661d-46d2-8d3c-2f4b79b7f07d.png)